test/cpp/api/fft.cpp

```diff
@@ -3,13 +3,6 @@
 #include <torch/torch.h>
 #include <test/cpp/api/support.h>
 
-
-// Tests that the fft function can be called as usual
-TEST(FFTTest, unclobbered_fft) {
-    auto t = torch::randn({64, 2}, torch::dtype(torch::kDouble));
-    torch::fft(t, 1);
-}
-
 // Clobbers torch::fft the function with torch::fft the namespace
 #include <torch/fft.h>
```

Comment:
Does fft.h need updates to prevent calling these functions from the C++ API, too?

Reply:
`fft.h` only deals with the fft module, the `torch::fft` functions was just generated by codegen so is gone completely. And actually as you point to this include, it's not needed any more since `torch.h` automatically includes the fft namespace as well now.